models/articles.model.js

```diff
@@ -10,3 +10,23 @@ exports.selectArticleById = (article_id) => {
       return rows;
     });
 };
+
+// All Articles with comment_count
+exports.getArticles = () => {
+  return db
+    .query(
+      `
+    SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY created_at DESC;
+    `
+    )
+    .then(({ rows }) => {
```

Comment:
If PSQL is correct this Promise.reject should never be reached.   So you don't need it.  All your PSQL errors should be caught by your 500 error handler in this case.